transit_odp/otc/querysets.py

```diff
@@ -381,6 +381,26 @@ class OperatorQuerySet(QuerySet):
     def add_service_count(self):
         return self.annotate(service_count=Count("services"))
 
+    def get_otc_organisation_name(self, licence_number):
+        from transit_odp.otc.models import Licence as OTCLicence
```

Comment:
Is there a specific reason for local imports?

Reply:
@pritish-ranjan01 These two models are only used within this function that's why I decided to put them here. Is it better to put them at the top of the file instead?

Reply:
Function level imports are usually discouraged because this could lead to repeated imports when the function is called multiple times. So, better to have it imported at top level.